### DISCUSSION TITLE
How to restart an executable when restarting i3

### DISCUSSION THREAD

**Discussion by TheKamboy** [regolith-linux/regolith-desktop#996] [Q&A] [answered]
I have xborders so that I have rounded corners. Is there any way I can restart it when I restart i3 or if there is a way to have xborders always on top? The reason why I'm asking is that when i3 restarts while still in the session, for when I edit a config, the windows start to go on top of the border.

**Comment by TheKamboy** [comment]
While I wait for an answer, I made a keybinding that allows me to manually restart xborders.

**Comment by kgilmer** [comment, marked as answer]
Hi @TheKamboy , I have not used `xborders` myself so not sure.  I believe you can achieve this with configuration changes to the `picom` config, if you're using that may be of interest.  Here is [an example](https://github.com/regolith-linux/regolith-look-extra/blob/main/usr/share/regolith-look/nord/picom.conf#L131).

**Reply by TheKamboy** [reply]
Good to know, I will check that out.
